### DISCUSSION TITLE
请打开网站 https://burn.hair/ ，接受用户协议然后继续使用本服务

### DISCUSSION THREAD

**Discussion by FHGZS** [BurnHair/docs#180] [技术支持]
请打开网站 https://burn.hair/ ，接受用户协议然后继续使用本服务？？ 也没找到在哪里接受用户协议

**Comment by BennyThink** [comment]
打开会自动弹窗的，你没有弹窗吗？那么可能要检查下广告屏蔽器，或者登出-登入，隐身模式，再看一下

**Reply by FHGZS** [reply]
现在可以了，某个插件导致屏蔽了。谢谢
